specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-09-30-preview/HybridCompute.json

```diff
@@ -3217,7 +3221,7 @@
         "Disabled"
       ],
       "x-ms-enum": {
-        "name": "SubscriptionStatus",
+        "name": "SoftwareAssuranceStatus",
```

Comment:
Since we are tracking if a customer is SA or not, can we not just say that.. 

isSoftwareAssuranceCustomer : true/false

(whatever the is right arm way to represent boolean)

Reply:
Changed it to a boolean value `isSoftwareAssuranceCustomer`